firebase/functions/calendarEventsbyUserId.ts

```diff
@@ -37,20 +42,30 @@ export async function getUserEvents(
     if (inviteeSnapshot.empty && createdSnapshot.empty) {
       return []
     }
-    let events1: CalendarEvents[] = []
-    let events2: CalendarEvents[] = []
+
+    let inviteeEvents: CalendarEvents[] = []
+    let creatorEvents: CalendarEvents[] = []
 
     if (date) {
-      events1 = inviteeSnapshot.docs
+      inviteeEvents = inviteeSnapshot.docs
         .filter((item) => item.data().eventStartTime >= date)
         .map((item) => ({ id: item.id, ...(item.data() as BaseEvents) }))
-      events2 = createdSnapshot.docs
+      
+        creatorEvents = createdSnapshot.docs
         .filter((item) => item.data().eventStartTime >= date)
         .map((item) => ({ id: item.id, ...(item.data() as BaseEvents) }))
+
+    }else{
+      inviteeEvents = inviteeSnapshot.docs
+        .map((item) => ({ id: item.id, ...(item.data() as BaseEvents) }))
+      
+        creatorEvents = createdSnapshot.docs
+        .map((item) => ({ id: item.id, ...(item.data() as BaseEvents) }))
     }
 
-    const mergeArray = [...events1, ...events2]
-    return mergeArray
+    const mergeArray: CalendarEvents[] = [...inviteeEvents, ...creatorEvents];
```

Comment:
I wonder, will it not be useful in the future to get invitee and creator events separate data streams? You can use that differentiation later?